content/docs/glossary/muxed-accounts.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Multiplexed Accounts
+order:
+---
+
+import { Alert } from "components/Alert";
+
+<Alert>
+
+  **Warning**: *This feature is experimental.*
+  Multiplexed accounts are not widely supported yet. Many wallets may not display a muxed account address and instead display the account ID address, ignoring the 64-bit ID. Adoption efforts are actively in progress.
+
+</Alert>
+
+
+A **multiplexed** (or **muxed**) **account** is an account that exists "on top of" a traditional Stellar account address. It combines the familiar `GABC...` address alongside a 64-bit integer ID in order to support differentiating multiple "virtual" accounts that share an underlying "real" account.
+
+They were introduced into the protocol in [CAP-21](https://stellar.org/protocol/cap-21) and their string representation is described in [SEP-23](https://stellar.org/protocol/sep-21). Muxed accounts have their own address format that starts with an `M` prefix. For example, from the account address `GA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVSGZ`, we can create two new muxed accounts with different IDs:
+
+  - `MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJUAAAAAAAAAAAACJUQ` has the ID `0`, while 
+  - `MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJUAAAAAAAAAABUTGI4` has the ID `420`.
+
+Both of these addresses, when used with one of the [supported operations][supported-ops], will act on behalf of their underlying `GA7Q...` address.
+
+With muxed accounts, we can *natively* support the ecosystem-level abstraction of account memo requirements (see [SEP-29](https://stellar.org/protocol/sep-29)), where the memo was often used to differentiate between customers backed by a single custodial account. Now, the embedded ID can be used for this distinction, and guides like [this one](../building-apps/setup-custodial-account.mdx) can be made superfluous.
+
+There are many other benefits to embedding this abstraction into the protocol:
+
+  - **"Share-ability"**: Rather than worrying about error-prone things like transaction memo IDs, you can just share the `M...` address.
+  - **SDK support**: The various [SDKs](../software-and-sdks) support this abstraction natively, letting you create, manage, and work with muxed accounts easily.
+  - **extensibility**: Using 64 bits for the integer ID embedded into each muxed account ensures that most abstractions can be encoded; it can be used to represent almost anything.
```

Comment:
This piece on extensibility is unclear to me, we made a mistake in making these IDs only 64bits and not arbitrary lengths, or at least 128bits because how M addresses are defined doesn't allow us to extend them. I might be misunderstanding the intent with discussing extensibility here. Could you share an example of what you mean?

Reply:
Honestly kinda phoned it in for this one; I wanted a >= 3 item list. The idea was that 64 bits should be plenty for any sort of "virtual encoding" users might want. Can you think of any other perks to include of muxed accounts instead of this item?

Reply:
Unfortunately 64 bits was in some ways a mistake and we can't actually encode any virtual encoding into it. 128bits would have been better, but here we are.

Opaque is a big benefit that helps with shareability and you could mention that.

Reply:
I thought of another perk; the bullet is now "efficiency." Let me know what you think!

Reply:
What did you mean by "opaque"?

Reply:
Efficiency sgtm. I think you can mention that virtual accounts are now first class citizens.

Opaque in the sense the muxed account is just a single address even though it contains the elements described in this doc, from much of the ecosystems pov it's just another account and what's inside it is irrelevant.

Reply:
Good call re: first class! I think the opaque notion is good but it's in some ways covered by the other bullets, so efficiency fits better here.